packages/hardcaml/hardcaml.1.1.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.1"
+maintainer: "andy.ray@ujamjar.com"
+homepage: "https://github.com/ujamjar/hardcaml"
+build: [
+  [make "all"]
+  [make "install"]
+]
+remove: [
+  [make "uninstall"]
+]
+depends: [
+  "ocamlfind"
+  "base-bytes"
+  "camlp4"
+  "js_of_ocaml"
+  "ctypes"
```

Comment:
do you have a minimum version of ctypes here? the interface has changed slightly in 0.3 and 0.4